### PR TITLE
On NoScript XSS info page, add link back to submission page

### LINKED ISSUE #4208: Add "« Back to submission page" link to NoScript docs
The `why-journalist-key` page in the Source Interface has a helpful "« Back to submission page" link at the bottom, the `disable-noscript-xss` page does not. We should have the link on both pages (or make other consistent UX changes to both pages), especially given that otherwise there's only a prominent "LOG OUT" link, and users may be disinclined to click the "Back" button in connection with forms.

# User Story

As a source, I want to know how I get back to the submission screen after learning about fun technical workarounds, so that I can successfully continue my submission.

### PR DESCRIPTION
**Pull request by rmol**
## Status

Ready for review

## Description of Changes

Adds a link back to the submission page at the bottom of the page explaining how to work around the NoScript cross-site sanitization upload problem.

Fixes #4208

## Testing

On the submission page, click the "Show me how" link to get to the instructions for working around the NoScript XSS upload problem. At the bottom of the page should be a "Back to submission page" link. Click it to verify that it takes you back to the source submission page.

### If you made changes to the server application code:

- [X] Linting (`make ci-lint`) and tests (`make -C securedrop test`) pass in the development container

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -119,6 +119,7 @@ ENV/
 
 # Emacs
 \#*#
+.emacs.desktop*
```
Comment (eloquence) [inline .gitignore]: Is that meant to be in this PR?

Comment (rmol) [reply .gitignore]: I can file another for it.

Comment (eloquence) [reply .gitignore]: Up to the maintainers but that would be my suggestion if it's unrelated. :)

Comment (rmol) [reply .gitignore]: Removed.

**Pull request merged**